plugins/cpp/parser/src/cppparser.cpp

```diff
@@ -358,6 +484,32 @@ bool CppParser::parse()
   return success;
 }
 
+void CppParser::initBuildActions()
+{
+  (util::OdbTransaction(_ctx.db))([&, this] {
+    for (const model::BuildAction& ba : _ctx.db->query<model::BuildAction>())
+      _parsedCommandHashes.insert(util::fnvHash(ba.command));
+  });
+}
+
+void CppParser::markAsModified(const model::FilePtr file_)
+{
+  auto inclusions = _ctx.db->query<model::CppHeaderInclusion>(
+    odb::query<model::CppHeaderInclusion>::included == file_->id);
+
+  for (auto inc : inclusions)
+  {
+    model::FilePtr loaded = inc.includer.load();
+    if(_ctx.fileStatus.count(loaded->path) == 0)
```

Comment:
space, and `!....count(...)` should express the same thing

Reply:
Fixed in 64cb407581662e78675baf916a95dba2b765b3e4.